lib/syskit/network_generation/logger.rb

```diff
@@ -129,7 +129,7 @@ def self.add_logging_to_network(engine, work_plan)
                     unless (logger_task = deployment.logger_task)
                         warn "deployment #{deployment.process_name} has no usable "\
                              "logger (default logger name would be "\
-                             "#{deployment.process_name}_Logger))"
+                             "#{deployment.process_name}_Logger)"
```

Comment:
I had fixed that, but my target was on master instead of 'transition-to-runkit' https://github.com/rock-core/tools-syskit/pull/336

every PR on tools-syskit (from our side) should target `transition-to-runkit`?

Reply:
Until we get the runkit branch merged, yes, I'm afraid.

It should merge cleanly, so I don't think it's an issue.